CHANGES.txt

```diff
@@ -4,8 +4,11 @@ Changes
 2.6 (unreleased)
 ----------------
 
-- Nothing changed yet.
-
+- compatibility for python 3
+- update code to work with python2 and python3
+- adjust egg to work with newer version of setuptools
```

Comment:
I don't know what "adjust egg" means.  Did you mean "adjust setup.py"?

It might be nice to say which version of setuptools it was that required changes, but if you don't know offhand, don't worry.